### PR TITLE
Add alifd__next via single package.json w/ npm auto-update

### PR DESCRIPTION
**Pull request by youluna**
Thank you for your awesome job! 

this PR is  for https://github.com/cdnjs/cdnjs/issues/13827

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required): https://github.com/alibaba-fusion/next
* Official website (optional, not the repository): https://fusion.design/
* NPM package url (optional): https://www.npmjs.com/package/@alifd/next
* License and its reference: MIT
* GitHub / Bitbucket popularity (required):
  * Count of stars: 2998
  * Count of watchers: 64
  * Count of forks: 286
* NPM download stats (optional):
  * Downloads in the last day: 455
  * Downloads in the last week: 2717
  * Downloads in the last month: 13,978

## Essential checklist

* [x] I'm the author of this library
  * [x] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR
* [x] The lib has notable popularity
  * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [x] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [ ] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [x] Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.
* [x] Pull request is sent from a non-master branch with a meaningful name.
* [x] Separate unrelated changes into different commits.
* [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [x] Close corresponding issue in commit message
* [x] Mention related issue(s), people in commit message, comment.

cc@bindoon

### CODE REVIEW

File: alifd__next/package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "alifd__next",
+  "filename": "next.min.js",
+  "version": "1.19.24",
```
Comment (MattIPv4) [inline alifd__next/package.json:4]:
```suggestion
```
Please remove the version attr, the auto-update bot will add this itself once it pulls in versioned files.

File: alifd__next/package.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  "description": "A configurable component library for web built on React.",
+  "homepage": "https://github.com/alibaba-fusion/next",
+  "keywords": [
+    "fusion",
+    "fu`sion design",
```
Comment (MattIPv4) [inline alifd__next/package.json:9]:
```suggestion
    "fusion design",
```
Typo?

File: alifd__next/package.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    "target": "@alifd/next",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      }
+    ]
+  }
```
Comment (MattIPv4) [inline alifd__next/package.json:44]:
```suggestion
  ]
```
Please remove the `autoupdate` attr, this is not the correct usage. Auto-updating with NPM is done via the `npmName` and `npmFileMap` attrs.

**Comment by youluna** [comment]
Thank you for your comment! 

I made this change via github site, since I even don‘t have 2G of free space on my computer  T_T 
Creating a new branch and commit the change is the most convenient way I can think of.

A new branch means a new PR, so I'll create another PR to fix this issues, I hope it won't cause you any trouble.

**Pull request closed without merging**